products/content-localization/README.md

```diff
@@ -18,20 +18,110 @@ Based on https://github.com/department-of-veterans-affairs/va.gov-team/issues/53
+### Translated Pages and CMS Menus (front end "sidebar")
+If a page has sidebar navigation on the front end and a Spanish and/or Tagalog translation for that page is added into Drupal, the new translated page(s) must have the same sidebar information in Drupal with the appropriate translations. Sidebar information is in Drupal under Menus and is in a hierarchical structure with a parent > child > grandchild (etc.) relationship.
+
+The gray background section of the sidebar on the front end shows the currently selected page and its child pages. When a translated page is added, the child pages (whether they are only in English or not), must be duplicated in the menu and associated with the translated parent.
+
+If the child pages are only in English, there is no need to add the grandchild pages to the translated menu hierarchy. This is because once you click on the English child page (from a translated parent), you are then using the English parent's menu hierarchy and already have information for the grandchildren.
+
+If any of the child pages of the new translated page also have Spanish and/or Tagalog translations, all grandchild links will need to be added as their parent is now a Spanish or Tagalog page and references the translated menu hierarchy.
+
+#### Example
+On [va.gov/disability/eligibility/](https://www.va.gov/disability/eligibility/), the page's sidebar menu in Drupal has a structure like the below.
+
+```
+- Eligibility
+  - Exposure to hazardous materials
+    - Specific environmental hazards
+    - Agent Orange
+    - Asbestos
+    - Mustard gas or lewisite
+    - Camp Lejeune water contamination
+    - Radiation exposure
+    - Project 112/SHAD
+    - Gulf War Illnesses SW Asia
+    - Gulf War Illnesses Afghanistan
+  - Former POWs
+  - PTSD
+  - Illnesses within 1 year after discharge
+  - Special claims
+    - Auto allowance and adaptive equipment
+    - Birth defects
+    - Clothing allowance
+    - Dental care
+    - Increase after surgery or cast
+    - Increase for time in hospital
+    - Prestabilization ratings
+    - Title 38 U.S.C. 1151 claims
+    - Unemployability
+```
+
+A translated version of this page ([va.gov/disability/eligibility-esp](https://www.va.gov/disability/eligibility-esp)) should have a menu item (`Elegibilidad`) as a sibling of `Eligibility`, and its 5 children (`Exposure...`, `Former POWs`, etc.) added and associated with `Elegibilidad`. If any of the 5 children are have Spanish translations, or any of the grandchildren, all children and grandchildren must be added and linked to their Spanish parent.
```

Comment:
If any of the 5 children are have >> If any of the 5 children have

Reply:
👍